.github/workflows/nodejs.yml

```diff
@@ -16,7 +16,7 @@ jobs:
     strategy:
       matrix:
         os: [macOS-latest, ubuntu-latest]
-        node-version: [10.x, 12.x]
+        node-version: [10.x]
```

Comment:
12.x suffered from [this](https://github.com/SimulatedGREG/electron-vue/issues/871).
Before investigating further on this issue, I believe we must upgrade our electron version since it's quite old now